src/extensions.jl

```diff
@@ -0,0 +1,13 @@
+isabelian(G::Group) = false
+issolvable(G::Group) = false
+isnilpotent(G::Group) = false
+isperfect(G::Group) = false
```

Comment:
Could it better to throw an error here? It feels like we are defining that every group is non-commutative etc., don't know if I would like that behavior. But maybe...

Reply:
Yeah, better an error - i.e. just do it like below and only provide a stub which can be documented but no actual implementation